cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/namespace/StorageProviderNamespaceAdminTest.java

```diff
@@ -144,7 +144,7 @@ public void testNSWithCustomLocation() throws Exception {
       file1.exists()));
     // but custom namespace location should still exists
     Assert.assertTrue(custom.exists());
-    custom.delete();
+    Assert.assertTrue(custom.delete());
```

Comment:
Good practice to clean up resources created in the test. If this fails, it may affect other tests in future.